server/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM openmicroscopy/omero-server:latest
+FROM openmicroscopy/omero-server:5.6
```

Comment:
Is there a reason to change this away from `latest`? It seems like it'll mean more work in future.

Reply:
Felt more correct to match the omego statement (which didn't get updated as it was in #5 -- will add that)